fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -1091,8 +1134,23 @@ LSTATUS __stdcall  RegGetValueFixup(
 #ifdef _DEBUG
         Log("[%d] RegGetValue:\n", RegLocalInstance);
 #endif
+
+        // If subkey is in redirected path, use it to get Registry Value
+        auto [updatedSubKey, keypath] = ReplaceRegistryQueryPath(&key, SubKey);
+        if (updatedSubKey)
+        {
+            if constexpr (psf::is_ansi<CharT>) 
```

Comment:
should we similarly retain ansi/wide char calls in RegOpenKeyEx and RegOpenKeyTransactedEx detours too?